### DISCUSSION TITLE
403 Error when trying to connect - API changes?!

### DISCUSSION THREAD

**Discussion by setiathome** [bisand/node-red-contrib-panasonic-comfort-cloud#27] [General]
Hello,

unfortunately the connection is not working. I am always getting the 403 ERROR. I tried to connect via [Homebridge](https://github.com/homebridge-panasonic-ac-platform/homebridge-panasonic-ac-platform/tree/master) which also works very well. It says here that there was a change in the API from Panasonic. They also added MFA in January 2024. Since the last change was 9 months ago, I assume that the error is somehow related to this?

Best regards.

**Comment by bisand** [comment]
Since I’ve started this API, my time to keep this library up-to-date has deminished, so please submit a pull request for the fix/improvements. I mighet have some time during the summer, but don’t hold your breath… sorry guys, but I think I need some help handelling these requests in the future. Hope you are up for it?!? 
Please feel free to take over the whole fepository, or feel free to submit a pull request.

thanks!
Best regards,
_An open source developer that tries to deliver upon the expectations_
Sorry!

**Comment by bisand** [comment]
I've replaced the underlying library with a more updated one. See if the new verison fixes your problems.
